src/system_simulation/server/test_models/linear_seesaw3.mo

```diff
@@ -0,0 +1,35 @@
+model linear_seesaw3
```

Comment:
Maybe you want to make a quick comment at the beginning of the .mo files what they are used for, or how they look, (maybe also link to a picture) to better understand what they are doing.